mr/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergSplit.java

```diff
@@ -79,12 +86,36 @@ public void write(DataOutput out) throws IOException {
     byte[] data = SerializationUtil.serializeToBytes(this.task);
     out.writeInt(data.length);
     out.write(data);
+
+    byte[] ioData = SerializationUtil.serializeToBytes(io);
```

Comment:
@rdblue @rdsr , I saw your comments in #985 about remove customized serialization. Now the `IcebergSplit` implements `org.apache.hadoop.mapred.InputSplit` which extends writable. So I keep serialization implementation. Does this make sense to you?

Reply:
I'm ok with that.